x-pack/plugins/observability/public/pages/overview/index.tsx

```diff
@@ -64,8 +67,8 @@ export function OverviewPage({ routeParams }: Props) {
   const { refreshInterval = 10000, refreshPaused = true } = routeParams.query;
 
   const relativeTime = {
-    start: routeParams.query.rangeFrom ?? timePickerTime.from,
-    end: routeParams.query.rangeTo ?? timePickerTime.to,
+    start: routeParams.query.rangeFrom || sharedTimeState.from || timePickerTime.from,
```

Comment:
Perhaps this would read better:
```suggestion
    start: routeParams.query.rangeFrom || timePickerSharedState.from || timePickerDefaults.from,
```

This way it's clear that the order is: 
1. url
2. shared state
3. defaults